src/common/types/value.cpp

```diff
@@ -927,6 +931,18 @@ string Value::ToString() const {
 	}
 }
 
+string Value::ToStringAsIs() const {
+	if (is_null) {
+		return "NULL";
+	}
+	switch (type_.id()) {
+	case LogicalTypeId::BLOB:
+		return Blob::ToStringAsIs(string_t(str_value));
```

Comment:
We can just directly return `str_value` here, no? That should simplify the code here.

Reply:
I reverted this change.

Reply:
Looks great!